sklearn/gaussian_process/kernels.py

```diff
@@ -1617,11 +1672,10 @@ class Matern(RBF):
 
 
 
-    where :math:`d(\\cdot,\\cdot)` is the Euclidean distance,
-    :math:`K_{\\nu}(\\cdot)` is a modified Bessel function and
-    :math:`\\Gamma(\\cdot)` is the gamma function.
-    See [1]_, Chapter 4, Section 4.2, for details regarding the different
-    variants of the Matern kernel.
+    where :math:`\\alpha` is the scale mixture parameter, :math:`l` is
+    the length scale of the kernel and :math:`d(\\cdot,\\cdot)` is the
+    Euclidean distance.
+    For advice on how to set the parameters, see e.g. [1]_.
```

Comment:
Wow! What happened here! Was the old docstring not correct or is there mathemagical hallucination going on?

It's an explanation of a formula's variables. _There is no \alpha mentioned here!_

In conclusion, the model here has gone off the rails!???